packages/client/shared/gqlIds/MeetingMemberId.ts

```diff
@@ -1,7 +1,6 @@
-
 const MeetingMemberId = {
-  join: (meetingId: string, userId: string) => `${userId}::${meetingId}`,
-  split: (id: string) => {
+  join: ({meetingId, userId}: {meetingId: string; userId: string}) => `${userId}::${meetingId}`,
```

Comment:
are you opposed to keeping the same API as before with 2 params? Typescript will still tell us which is which & then `MeetingMemberId.join` has the same signature as all the other gqlIds

Reply:
I kinda like passing in an object when two args are of the same type as it prevents us from mixing up the order. But I'm happy to keep things consistent and revert that change 👌